### ISSUE TITLE
flowforge/file-server:1.1.0 no matching manifest for linux/arm/v7

### ISSUE THREAD

**Issue by flecoufle** [flowforge/docker-compose#55]
### Current Behavior

it's not really a bug but a rather than a supported architecture request for armv7...

when I do
```
$ docker-compose up -d
Pulling file-server (flowforge/file-server:1.1.0)...
1.1.0: Pulling from flowforge/file-server
ERROR: no matching manifest for linux/arm/v7 in the manifest list entries
```

### Expected Behavior

have a way to create ourselves the image for this arch or have it on dockerhub

### Steps To Reproduce

clone repo
do the builds like in documentation
docker-compose up -d

### Environment

- FlowForge version:1.1.0
- Node.js version:not revelant
- npm version:not revelant
- Platform/OS:armv7/pi4
- Browser:not revelant

**Comment by hardillb** [comment]
Duplicate https://github.com/flowforge/docker-compose/issues/54

The quick fix is to add the 1.1.0 tag to the docker-compose file

```
  file-server:
    image: "flowforge/file-server:1.1.0"
    networks:
      - flowforge
    restart: always
```

You can grab the dockerfile and package.json from https://github.com/flowforge/helm/tree/main/file-server to build the container locally

**Comment by hardillb** [comment]
I will tag a new version with the tag in place this morning
